prov/verbs/configure.m4

```diff
@@ -19,7 +19,30 @@ AC_DEFUN([FI_VERBS_CONFIGURE],[
 				[],
 				[$verbs_PREFIX],
 				[$verbs_LIBDIR],
-				[verbs_ibverbs_happy=1],
+				[
+				echo -n "Checking if libibverbs is linkable by libtool... "
+				file=conftemp.$$.c
+				rm -f $file conftemp
+				cat > $file <<EOF
+/* Override any GCC internal prototype to avoid an error.
+   Use char because int might match the return type of a GCC
+   builtin and then its argument prototype would still apply.  */
+#ifdef __cplusplus
+extern "C"
+#endif
+char ibv_open_device ();
+int main ()
+{ return ibv_open_device (); }
+EOF
+				./libtool --mode=link --tag=CC $CC $CPPFLAGS \
+				$CFLAGS $file -o conftemp $LDFLAGS -libverbs \
+				2>&1 > /dev/null
+				status=$?
+				AS_IF([test $status -eq 0 && test -x conftemp],
+					[echo "yes"
```

Comment:
Use AC_MSG_RESULT instead of `echo`.  You also need a AC_MSG_RESULT for the `no` case.